tokio-util/src/time/delay_queue.rs

```diff
@@ -571,6 +771,49 @@ impl<T> DelayQueue<T> {
         }
     }
 
+    /// Shrink the capacity of the slab, which `DelayQueue` uses internally for storage allocation.
+    /// This function is not guaranteed to, and in most cases, won't decrease the capacity of the slab
+    /// to the number of elements still contained in it. To decrease the capacity to the size of
+    /// the slab use [`compact`]
+    /// This function can take O(n) time even when the capacity cannot be reduced or the allocation is
+    /// shrunk in place. Repeated calls run in O(1) though.
```

Comment:
If you want a line break, you have to include an empty line in the markdown. (Also, missing period.)
```suggestion
    /// the slab use [`compact`].
    ///
    /// This function can take O(n) time even when the capacity cannot be reduced or the allocation is
    /// shrunk in place. Repeated calls run in O(1) though.
```